e2e_test/batch/functions/trigonometric_funcs.slt.part

```diff
@@ -153,3 +153,138 @@ SELECT abs(atan2(pi(), -3.8) - 2.4507611878363194) < 0.000000000000001
 ----
 t
 
+query R
+SELECT abs(sinh(1) - 1.1752011936438014) < 0.000000000000001
```

Comment:
Equality of imprecise float is inherently hard 😢 FYI postgres regress test only test for 15 significant decimal digits for `double precision`, using the `extra_float_digits` config we do not support yet.

https://github.com/risingwavelabs/risingwave/blob/3091f1686ed40992768c5fbfec960b89b8c492f9/src/tests/regress/data/expected/float8.out#L640-L647

But we can manually adjust the precision requirement here, and we have no intention to do better than rust builtin `f64::sinh`...

What about
```
SELECT abs(sinh(1) - 1.17520119364380) < 1e-15;
-- or even 1e-14 if the ci machine is less capable
```

btw it is also a bit clearer to use the `1e-15` syntax rather than counting the zeros.